historian/src/lib.rs

```diff
@@ -35,8 +35,8 @@ fn record_match_event(
     events: EventStreamsHostBinding,
     msg: Vec<u8>,
 ) -> HandlerResult<()> {
-    logger.info("Recording match event")?;
     let evt: MatchEvent = serde_json::from_slice(&msg)?;
+    logger.info(&format!("Recording match event: {:?}", evt))?;
```

Comment:
You should be able to use `info!("....")` in actors now. This still requires the logging capability, but it's transparent to the developer.

Reply:
Where does the `info` macro come from in actors? I am on version `0.7.1`.